### PR TITLE
fixed biom equality unit tests

### PR DESCRIPTION
**Pull request by nbokulich**
nothing was wrong with the tests, only how `biom.table.descriptive_equality` was used — it returns a strings and does not raise an error on its own. This is now fixed.

### CODE REVIEW

**Comment by thermokarst** [comment]
@nbokulich, can you fix the failing travis runs here?

**Comment by nbokulich** [comment]
cannot fix: blocked by [this](https://github.com/knights-lab/SHOGUN/pull/21)

File: ci/recipe/meta.yaml [hunk 1]
```diff
@@ -14,14 +14,14 @@ build:
 
 requirements:
   build:
```
Comment (ebolyen) [inline ci/recipe/meta.yaml]:
```suggestion
   host:
```
New conda uses build for compile toolchains, it's kind of backwards compatible right now, but may as well update it.

File: q2_shogun/tests/test_shogun.py [hunk 2]
```diff
@@ -42,7 +42,9 @@ def test_nobunaga(self):
         taxa = shogun.actions.nobunaga(
             query=self.query, reference_reads=self.refseqs,
             reference_taxonomy=self.taxonomy, database=self.database)
-        taxa.taxa_table.view(biom.Table).descriptive_equality(self.taxatable)
+        taxa_table = taxa.taxa_table.view(biom.Table).sort()
+        report = taxa_table.descriptive_equality(self.taxatable)
+        self.assertIn('Tables appear equal', report, report)
```
Comment (ebolyen) [inline q2_shogun/tests/test_shogun.py:47]: I like this

**Review by ebolyen — CHANGES_REQUESTED** [review]
One change in the recipe, otherwise LGTM

**Comment by nbokulich** [comment]
thanks @ebolyen ! I committed your change

File: ci/recipe/meta.yaml [hunk 3]
```diff
@@ -13,15 +13,15 @@ build:
   script: make install
 
 requirements:
-  build:
-    - python 3.5*
+   host:
```
Comment (ebolyen) [inline ci/recipe/meta.yaml]:
```suggestion
  host:
```

**Comment by ebolyen** [comment]
@nbokulich, sorry I messed up the whitespace in the github editor :(

**Review by ebolyen — APPROVED** [review]

**Pull request merged**